content/en/sensitive_data_scanner/_index.md

```diff
@@ -119,28 +172,91 @@ You can create custom scanning rules using regex patterns to scan for sensitive
 {{% sds-scanning-rule %}}
 1. Click **Add Rule**.
 
-{{% /tab %}}
-{{< /tabs >}}
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
+[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
+{{< /collapse-content >}} 
 
 **Notes**:
 
-- Any rules that you add or update only affect data coming into Datadog after the rule was defined.
+- Any rules that you add or update affect only data coming into Datadog after the rule was defined.
 - Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
-- To turn off Sensitive Data Scanner entirely, set the toggle to **off** for each Scanning Group and Scanning Rule so that they are disabled.
+- After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.
+
+{{% /tab %}}
+{{% tab "Using the Agent" %}}
+
+A scanning rule determines what sensitive information to match within the data defined by a scanning group. You can add **up to 20** predefined scanning rules from Datadog's Scanning Rule Library. The Datadog Agent scans your data in your local environment during log collection, before logs are sent to the Datadog platform.
```

Comment:
This information is exactly the same as the one described below. I suggested to remove it here. The section is more explicit:
 ```suggestion
A scanning rule determines what sensitive information to match within the data defined by a scanning group. The Datadog Agent scans your data in your local environment during log collection, before logs are sent to the Datadog platform.
```